controller/DonationController.js

```diff
@@ -45,10 +45,16 @@ exports.handleFluttwerwaveRequests = catchAsyncError(async (req, res) => {
     await Donation.createDonation(payload);
   }
 
-  Mail.sendThankYouForDonation(
-    payload.data.customer.email,
-    payload.data.amount
-  );
+  try {
+    if (payload.data.customer.name.toLowerCase() !== "anonymous") {
```

Comment:
Can we write this as:

`if (!payload?.data?.customer?.name || payload?.data?.customer?.name.toLowerCase() !== 'anonymous`

Reply:
Asides the redundancy, there's the fact that we can't even send a successful request to Flutterwave without an email. Flutterwave will throw an error if there's no email, and our webhook will never get hit. 
There isn't really a situation where this function as a whole can get called without an email passed.

Reply:
Alright @Chiemezuo, that makes sense. Thanks for confirming that.